platforms/linux/riscv64-clang.toolchain.cmake

```diff
@@ -0,0 +1,27 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR riscv64)
+
+set(RISCV_CLANG_BUILD_ROOT /home/git/rvv-llvm/build CACHE PATH "Path to CLANG for RISC-V cross compiler build directory")
```

Comment:
Maybe it would be better to use paths based in `/opt` as default locations?

Reply:
Now the release version of RISC-V toolchain does not support latest RISC-V vector extension. We use the latest version of toochain installed by code in GitHub. And Clang does not support also. We use a Github version that under development. Should I still edit the path here?

Reply:
Are these paths hardcoded in the toolchain installation instruction?

Usually `/home` directory contains users' home directories, according to these lines your system have two users named `git` and `RISCV`. In some cases software can be installed in `/opt` directory, it is universal location and will not confuse readers.

Reply:
they are not hardcoded. It seems that my understanding of Linux path is not in place. Thanks for the guidance. I'll edit the default path here.